packages/esm-patient-chart-app/src/patient-chart/patient-chart.component.tsx

```diff
@@ -77,7 +77,7 @@ const PatientChart: React.FC = () => {
             </>
           )}
         </div>
-        <ActionMenu open={false} />
+        <ActionMenu open={active} />
```

Comment:
I'm a little confused on the logic here? The action menu is `open` if the workspace is active? It feels like we might've needed a different prop for this.

Reply:
@ibacher It seems I may have misinterpreted the original idea. The intention is to conceal the action menu on tablet devices when a workspace form is open, allowing for increased working space on smaller tablets. I will do a follow on PR to address it.